### ISSUE TITLE
[BUG] "See in original website" link incorrect

### ISSUE THREAD

**Issue by zapatistay** [FreshRSS/FreshRSS#5182]
As the topic says, all of a sudden the "see in original website" links for rss feeds stopped showing correct address. Instead, the address is "http://192.168.2.75:8080/i/" and that's it. I do not remember changing any settings lately. This used to work fine yesterday.

Take a look at the image and see that even though the mouse is on the title, which should give a link to the original website, you can see on the left bottom that the address is my self-hosted freshrss website.

[https://imgur.com/a/rCWV6Rm](url)

**Comment by Alkarex** [comment]
Hello,
Is there the same issue on all your feeds?
Could you try to disable any FreshRSS extension you might have? And also try in another Web browser?
Please give details about your FreshRSS setup (version, how you installed it, type of platform...)

**Comment by zapatistay** [comment]
This problem is prevalent in all other feeds. The only thing I did was play with the extension.php file of the Invidious Video Feed extension. I modified the file such that it would work with http instead of the default expected https. This is because I setup my own local invidious server just to serve myself and I need it to work over http whereas the extension expects https. 

Aaaand yes when I disabled the Invidious Video Feed extension, the problem went away.  How is there a relation between the two ? Why is the Invidious Video Feed extension having an effect on other feeds ? I really want to get it to work locally.
